axelrod/tests/strategies/test_darwin.py

```diff
@@ -15,29 +15,45 @@ class TestDarwin(TestPlayer):
         'stochastic': False,
         'makes_use_of': set(),
         'long_run_time': False,
-        'inspects_source': False,
+        'inspects_source': True,
         'manipulates_source': False,
         'manipulates_state': True
     }
 
+    def setUp(self):
+        """Each test starts with a fresh genome."""
+        self.player.genome = [C]
+        super(TestDarwin, self).setUp()
+
+    def test_setup(self):
+        player = self.player()
+        self.assertEqual(player.genome, [C])
+        self.assertEqual(player.history, [])
+
     def test_strategy(self):
         p1 = self.player()
-        p2 = axelrod.Cooperator()
-        self.assertEqual(p1.strategy(p2), C)  # Always cooperate first.
-        for i in range(10):
-            p1.play(p2)
-        self.assertEqual(p1.strategy(p2), C)
+        p1.reset()
+        self.versus_test(axelrod.Cooperator(), [(C, C)] * 5)
```

Comment:
We have been going with `self.versus_test(Cooperator(), expected_actions=actions)` which (sorry) I'd prefer :) :+1: